tools/wptserve/tests/functional/test_pipes.py

```diff
@@ -126,6 +126,11 @@ def test_sub_fs_path(self):
 """ % {"root": root, "sep": os.path.sep}
         self.assertEqual(resp.read(), expected.encode("utf8"))
 
+    def test_sub_headers(self):
```

Comment:
./wptserve/tests/functional/test_pipes.py:129:5: F811 redefinition of unused 'test_sub_headers' from line 81

Probably call this `test_sub_header_or_default`?